setup.cfg

```diff
@@ -1,2 +1,103 @@
+long_description_content_type = text/x-rst; charset=UTF-8
+description_file = file:README.md
+description_content_type = text/markdown; charset=UTF-8
+
+url = https://github.com/CleanCut/green
+
+classifier =
+    Development Status :: 5 - Production/Stable
+    Environment :: Console
+    Intended Audience :: Developers
+    License :: OSI Approved :: MIT License
+    Natural Language :: English
+    Operating System :: MacOS :: MacOS X
+    Operating System :: Microsoft :: Windows
+    Operating System :: POSIX
+    Operating System :: POSIX :: Linux
+    Operating System :: Unix
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Programming Language :: Python :: 3.11
+    Programming Language :: Python :: 3 :: Only
+    Topic :: Software Development :: Libraries
+    Topic :: Software Development :: Quality Assurance
+    Topic :: Software Development :: Testing
+    Topic :: Utilities
+
+keywords =
+    nose, nose2, trial, pytest, py.test, tox, green,
+    tdd, test, tests, functional, system, unit, unittest,
+    color, tabular, clean, red, rednose, regression, runner,
+    integration,smoke, white, black, box, incremental, end,
+    end-to-end, sanity, acceptance, load, stress, performance,
+    usability, install, uninstall, recovery, security,
+    comparison, alpha, beta, non-functional, destructive,
+    accessibility, internationalization, i18n, localization, l10n,
+    development, a/b, concurrent, conformance, verification,
+    validation, quality, assurance, ad-hoc, agile, api,
+    automated, all, pairs, pairwise, boundary, value, branch,
+    browser, condition, coverage, dynamic, exploratory,
+    equivalence, partitioning, fuzz, gui, glass, gorilla,
+    interface, keyword, penetration, retesting, risk, based,
+    scalability, soak, volume, vulnerability
+
+
+
+test_suite='green.test'
+
+# TODO: find a way to get $version to work here.
+# download_url = https://github.com/CleanCut/green/tarball/$version
+
+[options]
+python_requires = >=3.5
+# The file:requirements.txt syntax is not compatible until Python 3.7.
+# install_requires = file:requirements.txt
+install_requires =
+    colorama
+    coverage
+    lxml
+    unidecode
+include_package_data = True
+packages = find:
+
+[options.package_data]
+green = VERSION, shell_completion.sh
+
+
+[options.entry_points]
+console_scripts =
+    green = green.cmdline:main
+    green3 = green.cmdline:main
+    green3.5 = green.cmdline:main
+    # TODO: Figure out a way to not hardcode the versioned console scripts.
+    # 'green%d = green.cmdline:main' % sys.version_info[:1],    # green3
+    # 'green%d.%d = green.cmdline:main' % sys.version_info[:2], # green3.5 etc.
```

Comment:
This part needs to be completed. We can't leave the minor version number hard-coded.

Reply:
I understand ... I just do not know if this is possible with `setup.cfg` and/or `pyproject.toml`. If anyone here has some insight into it that would help.

Reply:
🤔 Installing Python packages globally was still mostly the norm when I created green in 2014, so it was quite helpful to have the `greenX.Y`-style scripts to deal with having multiple versions of Python installed.

Now that using virtualenv is so common, it's probably not necessary.

Let's go ahead and just delete both the `green3.5 = ...` and `green3 = ...` lines, leaving us with just `green`.

Reply:
I've removed the versioned entry points. Note that you might want to wait for the other PR to be merged so that the `black` changes are applied to the main branch.